conda_recipes/maya-vray-2025/README.md

```diff
@@ -0,0 +1,22 @@
+# V-Ray conda build recipe
+Use this recipe to create a V-Ray for Maya conda package to use with AWS Deadline Cloud. Conda packages let you customize the software you can use with your Deadline Cloud deployment. Read more about how to host a conda channel for these custom conda packages below.
+
+## Download the archive file
+- Download the `vray_adv_62002_maya2025_rhel8` full download file from [Chaos](https://download.chaos.com/downloads/23688/vray-maya-2025-62002-adv)\
+**_NOTE:_** Need to have a Chaos account to access the link
+
+## Build Conda Package
+- place it in the `conda_recipes/archive_files` directory in your git clone of the
+[deadline-cloud-samples](https://github.com/aws-deadline/deadline-cloud-samples) repository for
+submitting package build jobs.
+
+- Follow this [README](https://github.com/aws-deadline/deadline-cloud-samples/blob/mainline/conda_recipes/README.md) to submit a job building conda package.
+
+## Requirement
```

Comment:
This section repeats content from the `## Build Conda Package` section. Is there a way we can make the overall document more cohesive?